package.json

```diff
@@ -82,6 +82,7 @@
     "yeoman-generator": "^1.0.1"
   },
   "devDependencies": {
+    "@polymer/tools-common": "^1.0.1",
```

Comment:
hmm okay, the stakes are low enough that I'm willing to try. But my concern is still that we're giving ourselves a footgun to make small changes to the tslint file and version them as patches/minors, not realizing that they could remotely break other repos. If a human is still needed to manually upgrade major versions, then there's no extra work added by the more stable `=` vs. the more liberal `^`.